docs/spec/spec.md

```diff
@@ -1,23 +1,23 @@
 # Specification: Ballerina PostgreSQL Library
 
-_Owners_: @daneshk @niveathika  
-_Reviewers_: @daneshk  
-_Created_: 2022/01/14   
-_Updated_: 2022/01/14  
+_Owners_: @daneshk @niveathika
+_Reviewers_: @daneshk
+_Created_: 2022/01/14 
+_Updated_: 2022/01/14
```

Comment:
```suggestion
_Owners_: @daneshk @niveathika  
_Reviewers_: @daneshk  
_Created_: 2022/01/14  
_Updated_: 2022/01/14  
```

Else this will not be formatted properly

Reply:
Fixed in d4ddb80